lib/matplotlib/container.py

```diff
@@ -148,6 +148,66 @@ def __init__(self, lines, has_xerr=False, has_yerr=False, **kwargs):
         super().__init__(lines, **kwargs)
 
 
+class PieContainer:
+    """
+    Container for the artists of pie charts (e.g. created by `.Axes.pie`).
+
+    You can access the patches and further parameters by the attributes.
+
+    Attributes
+    ----------
+    patches : list of `~matplotlib.patches.Wedge`
+        The artists of the pie wedges.
+
+    values : tuple of int or float
+        The data that the pie is based on.
+
+    fracs : tuple of float
+        The fraction of the pie that each wedge represent.
+
+    texts : list of list of `~matplotlib.text.Text`
+        The artists of any labels on the pie wedges, except those added with the
+        *autotexts* parameter of `~.Axes.pie`.  Each inner list has one
+        text label per wedge.
```

Comment:
Should we keep the autotexts in texts. That leads to the same `pc.texts` for 

`pc = pie(..., auto_pct=...)` and `pc = pie(...); pie_label(pc, "{frac:.1%}")`, which feels more consistent. In that case `autotexts` is an implementation detail only used in `__getitem__`.